src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -116,10 +153,21 @@ def check_get_timestamps(self):
 
         assert len(timestamps) != 0
 
-    def check_align_starting_time_internal(self):
-        """Ensure that internal mechanisms for shifting timestamps given a starting time match expectation."""
+    def check_interface_align_timestamps(self):
+        """Ensure that internal mechanisms for the timestamps getter/setter work as expected."""
         fresh_interface = self.data_interface_cls(**self.test_kwargs)
-        unaligned_timestamps = fresh_interface.get_timestamps()
+        unaligned_timestamps = fresh_interface.get_original_timestamps()
+
+        aligned_timestamps = unaligned_timestamps + 1.23 + np.random.random(size=unaligned_timestamps.shape)
```

Comment:
I think we should make all the uses of random deterministic in tests. We should use np.random.default_rng` which is the recommendation from numpy:

https://numpy.org/doc/stable/reference/random/generator.html

Reply:
Sure thing, good call

Will adjust that in other tests too

Reply:
Fixed